app/namespace.go

```diff
@@ -57,6 +60,38 @@ var (
 		"us-east-1",
 		"us-west-2",
 	}
+	sinkNameFlag = &cli.StringFlag{
+		Name:     "sink-name",
+		Usage:    "sink name",
+		Required: true,
+	}
+	sinkEnabledFlag = &cli.BoolFlag{
+		Name:  "enabled",
+		Usage: "Whether the export sink is enabled",
+	}
+	sinkTypeFlag = &cli.StringFlag{
+		Name:     "type",
+		Usage:    "The type of the export sink",
+		Required: true,
+	}
+	sinkAssumedRoleFlag = &cli.StringFlag{
```

Comment:
is this the same terminology we are using for the role ARN in cloud ui/docs for audit log? We should probably try and align the same words